pkg/stub/handler.go

```diff
@@ -97,7 +102,13 @@ func (h *Handler) syncIngressUpdate(ci *ingressv1alpha1.ClusterIngress) error {
 	err = sdk.Create(ds)
 	if err == nil {
 		logrus.Infof("created router daemonset %s/%s", ds.Namespace, ds.Name)
-	} else if !errors.IsAlreadyExists(err) {
+	} else if errors.IsAlreadyExists(err) {
+		err = sdk.Update(ds)
```

Comment:
How do we know this is an update event and not resync event? (may be we want to update anyway during resync?)

Reply:
When @Miciah PR gets merged we will know when there's an update. I think this pattern is good for now.